package/src/external/reanimated/moduleWrapper.ts

```diff
@@ -0,0 +1,33 @@
+import { useMemo } from "react";
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+let Reanimated: any;
+
+try {
+  Reanimated = require("react-native-reanimated");
+} catch (e) {
+  // Ignore
+}
+
+export const HAS_REANIMATED = Reanimated !== undefined;
+
+function throwOnMissingReanimated() {
+  throw new Error(
+    "Reanimated was not found, make sure react-native-reanimated package is installed if you want to use \
+    react-naitve-skia's integration layer API."
+  );
+}
+
+export const useSharedValue =
```

Comment:
the five function definition below should have their own function signatures, this is because in this file Reanimated is typed as any.